packages/sanity/src/core/preview/createGlobalListener.ts

```diff
@@ -27,37 +24,9 @@ export function createGlobalListener(client: SanityClient) {
       },
     )
     .pipe(
-      share({
-        resetOnRefCountZero: () => timer(2000),
+      shareReplayLatest({
+        predicate: (event) => event.type === 'welcome' || event.type === 'reconnect',
```

Comment:
would `filter` be better than `predicate` here?

Reply:
I think `predicate` makes sense, given filter is a rxjs function, calling it filter here could miss lead to think it's the filter function. 

And in the `filter` function in rxjs they define the callback as `predicate`, so I consider it follows the convention. 

> [filter](https://rxjs.dev/api/index/function/filter)<T>(predicate: (value: T, index: number) => boolean, thisArg?: any)